jsonnet/main.jsonnet

```diff
@@ -284,17 +284,14 @@ local inCluster =
         version: $.values.common.versions.thanos,
       },
       thanosRuler: $.values.thanos {
-        name: 'user-workload',
+        name: 'thanos-ruler',
+        crName: 'user-workload',
         namespace: $.values.common.namespaceUserWorkload,
         replicas: 2,
-        labels: {
-          'app.kubernetes.io/name': 'user-workload',
-        },
-        selectorLabels: {
-          'app.kubernetes.io/name': 'thanos-ruler',
-          'thanos-ruler': 'user-workload',
-        },
+        serviceMonitor: true,
```

Comment:
Should we move it to `jsonnet/components/thanos-ruler.libsonnet` directly? I feel that it would be clearer since it's not something that we need to customize.
I've got the same question for `volumeClaimTemplate` below.